### PR TITLE
created ProgressBar component

### PR DESCRIPTION
**Pull request by HelenDrug**
Created Progress Bar component with React, added demo and document description

### CODE REVIEW

**Comment by ashispavan** [comment]
@HelenDrug Can you please resolve the conflicts and commit? Thanks!

**Comment by HelenDrug** [comment]
Wish I knew how to do that :/

**Comment by Caleb335** [comment]
@ashispavan i don't think there are any available conflicts in this PR, or have they been resolved? 

Screenshot_20200630-233416

File: src/components/ComponentList/ComponentList.jsx [hunk 1]
```diff
@@ -1,3 +1,7 @@
+
+
+/*Add your component name to the array */
+const components = ['Simple Button', 'Heading', 'Progress Bar']
```
Comment (madhuni) [inline src/components/ComponentList/ComponentList.jsx:4]: Can we declare our constants after all the imports?

File: src/demos/ProgressBarDemo.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    return (<>
+        <ReactMarkdown source={markdown} />
+        {testData.map((item, idx) => (
+        <ProgressBar key={idx} bgColor={item.bgColor} completed={item.completed} />
+      ))}
+    </>
+    )
+}
+
+export default ProgressBarDemo
```
Comment (madhuni) [inline src/demos/ProgressBarDemo.jsx:30]: Please add the newline at the end of the file.

**Comment by madhuni** [comment]
@HelenDrug, please run the prettier so that the code is formatted properly. Other changes look good to me.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,2 @@
+{
+}
```
Comment (madhuni) [inline .vscode/settings.json:2]: I think we should not push the `.vscode/settings.json` to the repo. I will suggest adding this folder into your `.gitignore` file.

**Comment by HelenDrug** [comment]
Yay! Thank you so much for guidance here :) this was very useful to me

**Pull request merged**